packages/web/src/shared/components/form/ComboboxInput.tsx

```diff
@@ -111,6 +50,13 @@ const BaseComboboxInput = forwardRef<HTMLInputElement, BaseComboboxInputProps>(
     const [normalizedInputValue, setNormalizedInputValue] = useState('');
     const [filteredItems, setFilteredItems] = useState<ComboboxItem[]>(items);
 
+    const formContext = useFormContext();
+    const hasErrors = !!(
+      formContext &&
+      name &&
+      formContext.getFieldState(name).error
+    );
```

Comment:
We're using this pattern a couple places. Could we put it in a helper function that accepts `formContext` and `name` as parameters?

Reply:
This can be simplified to the following, which I don't think is worth abstracting into it's own hook
```
const hasErrors = !!(name && formContext.getFieldState(name).error);
```